kennyedward/QuickCredit
# #165984426 Implement Landing page

#### What does this PR do?
Add a landing page and links to the loan application to improve the user and admin experience

#### Description of Task to be completed?
Have this URL showing the landing page
```https://kennyedward.github.io/QuickCredit/UI/```

#### How should this be manually tested?
Navigate to ```https://kennyedward.github.io/QuickCredit/UI/```
-  click on Apply to login as a user
- login with any credentials
-  wait for ```10seconds``` to get verification alert
-  you should be able to apply for a loan
- click on  **Apply for loan** button
- Input ```Amount``` and ```tenure``` in the form fields
-  wait for a notification that your application was successful 

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
#165984426,  #165331684, #165331828, #165331699, #165331663

#### Screenshots (if appropriate)
![new landing](https://user-images.githubusercontent.com/18053279/57774713-25933880-7713-11e9-966a-d5845a52a4b6.jpg)
![loanApply](https://user-images.githubusercontent.com/18053279/57774835-7a36b380-7713-11e9-94f8-451785728d2d.JPG)

#### Questions:
Nil
